agenttest.py

```diff
@@ -438,9 +438,18 @@ def setUpClass(cls):
 
     def test_end_to_end(self):
         """Run end to end ping test"""
-        failures = end_to_end_test()
+        options = dict(nohup=0)
+        envs = dict(config_stat_reload=0)
+        failures = end_to_end_test(options, envs)
```

Comment:
should just be something like `end_to_end_tests(nohup=0, config_stat_reload=True)`